data/tech/aria/navigation_role.json

```diff
@@ -9,6 +9,39 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#navigation"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_role",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "div[role=\"navigation\"]",
+      "operation_modes": [
+        "sr/reading",
+        "sr/interaction"
+      ]
+    },
+    {
+      "id": "convey_name",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "div[role=\"navigation\"]",
+      "operation_modes": [
+        "sr/reading",
+        "sr/interaction"
+      ]
+    },
+    {
+      "id": "convey_boundaries",
+      "css_target": "div[role=\"navigation\"]"
```

Comment:
Similar comment about making the selector less specific

Reply:
Updated